src/components/players/players-others/players-others.tsx

```diff
@@ -25,14 +26,11 @@ export const PlayersOthers: FC<Props> = ({
 }) => {
   const [isLoadingVotes, setIsLoadingVotes] = useState(false);
   const [selectedIds, setSelectedIds] = useState<string[]>([]);
+  const [viewResults, setViewResults] = useState<string>("1");
```

Comment:
Magic string. What is this? If it's a character id make it of `CharacterId` type so we know it's not a magic string but a literal

Reply:
And also if it's an id please find a better name for the variable e.g. `resultsCharacterId`